src/dex/camelot/camelot.ts

```diff
@@ -205,8 +213,58 @@ export class Camelot
 
     this.routerInterface = new Interface(ParaSwapABI);
     this.exchangeRouterInterface = new Interface(UniswapV2ExchangeRouterABI);
+
+    this.factoryInst = new UniswapV2Factory(
+      dexHelper,
+      dexKey,
+      factoryAddress,
+      this.logger,
+      this.onPoolCreatedDeleteFromNonExistingSet,
+    );
+  }
+
+  private _constructPoolKey(token0: string, token1: string) {
+    const [_token0, _token1] =
+      token0.toLowerCase() < token1.toLowerCase()
+        ? [token0, token1]
+        : [token1, token0];
+
+    const poolKey = `${token0}-${token1}`.toLowerCase();
```

Comment:
@mwamedacen '${_token0}-${_token1}'.toLowerCase()?

Reply:
I kept same convention as in the code

Reply:
@mwamedacen I meant it seems to me that you used incorrect vars. From what I see it should be `_token0` instead of `token0` and `_token1` instead of `token1`. In other case I don't understand why you need tokens sorting above.

Reply:
good catch !